rescript/dereplicate.py

```diff
@@ -139,7 +140,11 @@ def _dereplicate_taxa(taxa, raw_seqs, derep_seqs, uc, mode):
         if mode == 'lca':
             derep_taxa = derep_taxa.apply(lambda x: ';'.join(
                 _find_lca([y.split(';') for y in x]))).to_frame()
-            # find majority taxon within each cluster
+        # find majority superset LCA within each cluster
+        if mode == 'super':
```

Comment:
should be `elif`, not that it changes the functionality (since there's only one mode!) but it helps clarify

Reply:
oops! right!